Remotion/ObjectBinding/Web.UnitTests/UI/Controls/Validation/BocListValidationResultDispatchingValidatorTest.cs

```diff
@@ -37,10 +37,22 @@ public void RefreshErrorMessage_WithoutValidationFailures_ReturnsMessageFromReso
       validationFailureRepoStub
           .Setup(_ => _.GetUnhandledValidationFailuresForBocList(true))
           .Returns(Array.Empty<BocListValidationFailureWithLocationInformation>());
+
+      var sequence = new MockSequence();
```

Comment:
We have completely removed MockSequence because it is hard to work with (it hides calls etc).

For testing ordering, we use VerifiableSequence. In case when there's multiple calls with the same arguments, we use 
`mock.SetupSequence(mock => mock.Read())
          .Returns(true)
          .Returns(false);
`

Here, you can't assert ordering between calls, though.

From reading the tests, I think the SetupSequence/Returns semantic is what you want.